README.md

```diff
@@ -1,116 +1,18 @@
-Chef Supermarket Omnibus project
+Chef Supermarket Omnibus project (deprecated)
 ================================
 
-This project creates full-stack platform-specific packages for
-[Chef Supermarket](https://supermarket.getchef.com).
+DO NOT SUBMIT PATCHES OR CONTRIBUTE TO THIS REPOSITORY.
 
-Issues for this project can be opened and tracked from the
-[Supermarket GitHub Issues page](https://github.com/opscode/supermarket/issues).
+The omnibus project definition for Supermarket used to live in this repo. It has
+been migrated into [Supermarket's primary git repository](https://github.com/chef/supermarket)
+under [an omnibus directory](https://github.com/chef/supermarket/tree/master/omnibus).
```

Comment:
@nellshamrell Buried in the screenfuls of red, the README contains links to consolidation repo and omnibus directory within it.